definition/data/sheets/CaseEvent.json

```diff
@@ -39,6 +39,7 @@
   {"CaseTypeID": "MoneyClaimCase", "ID": "PinGenerationOperations", "Name": "BulkPrint Defendant & Staff ", "Description": "Bulk Print, Defendant & Staff notification", "DisplayOrder": 1, "PreConditionState(s)": "*", "PostConditionState": "*", "SecurityClassification": "Public"},
   {"CaseTypeID": "MoneyClaimCase", "ID": "SendingRPA", "Name": "RPA Request", "Description": "RPA Request", "DisplayOrder": 1, "PreConditionState(s)": "*", "PostConditionState": "*", "SecurityClassification": "Public"},
   {"CaseTypeID": "MoneyClaimCase", "ID": "SealedClaimUpload", "Name": "Sealed claim uploaded", "Description": "Sealed claim uploaded", "DisplayOrder": 1, "PreConditionState(s)": "*", "PostConditionState": "*", "SecurityClassification": "Public"},
+  {"CaseTypeID": "MoneyClaimCase", "ID": "ReviewOrderUpload", "Name": "Review order uploaded", "Description": "Review order uploaded", "DisplayOrder": 1, "PreConditionState(s)": "*", "PostConditionState": "*", "SecurityClassification": "Public"},
```

Comment:
Is this a request document thats uploaded? If yes, can you kindly label it more appropriately like Review order request uploaded

Reply:
👍